lambda/logserver/logserver.go

```diff
@@ -149,7 +156,12 @@ func (ls *LogServer) handler(res http.ResponseWriter, req *http.Request) {
 }
 
 func Start() (*LogServer, error) {
-	return startInternal(defaultHost)
+	// This should only be used for testing purposes
+	host, hostOverride := os.LookupEnv("NEW_RELIC_LOG_SERVER_HOST_OVERRIDE")
```

Comment:
Can we push this into the configuration object, so that we don't have env vars everywhere?